paperqa/agents/task.py

```diff
@@ -549,13 +550,14 @@ async def _evaluate_answer(self) -> dict:
             "answer2": self.human_answer if pqa_answer_index == 1 else pqa_answer,
         }
 
-        result = await pairwise_eval_llm.run_prompt(
-            prompt=lfrqa_prompt_template,
-            data=data,
-            system_prompt=lfrqa_system_prompt,
+        result = await pairwise_eval_llm.call_single(
+            messages=[
+                Message(role="system", content=lfrqa_prompt_template),
+                Message(role="user", content=lfrqa_system_prompt.format(**data)),
```

Comment:
```suggestion
                Message(role="system", content=lfrqa_system_prompt),
                Message(role="user", content=lfrqa_prompt_template.format(**data)),
```

@maykcaldas you got it backwards in two ways

- You have "system prompt" being given `"user"` role
- You have `.format` on the system prompt, but it's the user prompt that is the template